src/main/kotlin/org/rust/ide/docs/RsDocumentationProvider.kt

```diff
@@ -118,8 +127,15 @@ private fun RsDocAndAttributeOwner.signature(usePreTag: Boolean): String {
             retType?.generateDocumentation(buffer)
             listOf(buffer.toString()) + whereClause?.documentationText.orEmpty()
         }
-        // All these types extend RsTypeBearingItemElement and RsGenericDeclaration interfaces
```

Comment:
Let's replace mention about `RsTypeBearingItemElement` with `RsItemElement` - currently we use cast to `RsItemElement`.

Reply:
Done